packages/terra-form/src/Select.scss

```diff
@@ -0,0 +1,11 @@
+@import '~terra-mixins';
+
+.terra-Form-select {
+  background-color: #fff;
+  border: #dedfe0 solid 1px;
+  border-radius: 0.25em;
+  font-size: 1em;
+  line-height: 1.25;
+  padding: 0.178em;
+  width: 100%;
```

Comment:
The height on select is 22px whereas the height on the text input is 28px. Should the height be the same as the text input (https://github.com/cerner/terra-core/blob/master/packages/terra-form/src/Input.scss#L7)?